### PR TITLE
Update pray_times_calculator.py

### PR DESCRIPTION
**Pull request by laithalsunni**
Added the rest of the collection methods

### CODE REVIEW

**Comment by laithalsunni** [comment]
Please add the new calculation methods I may update the documents for home assistant too

File: prayer_times_calculator/pray_times_calculator.py [hunk 1]
```diff
@@ -5,10 +5,19 @@
 
 class PrayerTimesCalculator:
     CALCULATION_METHODS = {
+        'shia': 0, 
         'karachi': 1,
         'isna': 2,
         'mwl': 3,
-        'makkah': 4
+        'makkah': 4,
+        'egas': 5,
```
Comment (uchagani) [inline prayer_times_calculator/pray_times_calculator.py]: please change this to `egypt`

File: prayer_times_calculator/pray_times_calculator.py [hunk 2]
```diff
@@ -5,10 +5,19 @@
...
+        'igut': 6,
```
Comment (uchagani) [inline prayer_times_calculator/pray_times_calculator.py:14]: change this to `tehran`

File: prayer_times_calculator/pray_times_calculator.py [hunk 3]
```diff
@@ -5,10 +5,19 @@
...
+        'gulf': 7,
+        'kuwait': 8,
+        'qatar': 9,
+        'muss': 10,
```
Comment (uchagani) [inline prayer_times_calculator/pray_times_calculator.py]: `singapore`

File: prayer_times_calculator/pray_times_calculator.py [hunk 4]
```diff
@@ -5,10 +5,19 @@
...
+        'uoisdf': 11,
```
Comment (uchagani) [inline prayer_times_calculator/pray_times_calculator.py]: `france`

File: prayer_times_calculator/pray_times_calculator.py [hunk 5]
```diff
@@ -5,10 +5,19 @@
...
+        'dibt': 12
```
Comment (uchagani) [inline prayer_times_calculator/pray_times_calculator.py]: `turkey`

**Comment by laithalsunni** [comment]
I’ll get on it as soon as I get home.

File: prayer_times_calculator/pray_times_calculator.py (hunk 5 above)
Comment (laithalsunni) [reply prayer_times_calculator/pray_times_calculator.py]: All Done

**Review by laithalsunni — COMMENTED** [review]
all done

**Comment by uchagani** [comment]
@laithalsunni you forgot to change `igut` to `tehran`

**Comment by uchagani** [comment]
also @laithalsunni have you verified all of these calculation methods work?  I understand it's part of their documentation but when you hit the calculation methods endpoint (http://api.aladhan.com/v1/methods) not all of them are returned.  For example, `Egyptian General Authority of Survey` is not returned as a valid calculation method.

Please verify the IDs are correct for the ones that work and let's only include the ones that work in this pull request.

**Comment by uchagani** [comment]
@laithalsunni please verify the IDs are correct.  It looks like both in their documentation and http://api.aladhan.com/v1/methods they skip a number in the middle of their list somewhere.

**Comment by uchagani** [comment]
looks like `tehran` should be `7` and then increment from there.  ID of `6` doesn't exist.

**Comment by uchagani** [comment]
this was implemented in #4

**Pull request closed without merging**